src/transformers/models/auto/tokenization_auto.py

```diff
@@ -290,7 +290,13 @@
             ("wav2vec2", ("Wav2Vec2CTCTokenizer", None)),
             ("wav2vec2-conformer", ("Wav2Vec2CTCTokenizer", None)),
             ("wav2vec2_phoneme", ("Wav2Vec2PhonemeCTCTokenizer", None)),
-            ("whisper", ("WhisperTokenizer" if is_sentencepiece_available() else None, None)),
+            (
+                "whisper",
+                (
+                    "WhisperTokenizer" if is_sentencepiece_available() else None,
```

Comment:
Why is `sentencepiece` here @ArthurZucker, Whisper tokenizer files do not depend on sentencepiece at all.

Reply:
I removed the dependency, if not desired I will revert.